internal/permutations/kensler.go

```diff
@@ -0,0 +1,88 @@
+package permutations
+
+import (
+	"crypto/rand"
+	"fmt"
+	"math"
+	"math/big"
+)
+
+/*
+
+Andrew Kensler, a researcher at Pixar, introduced an interesting technique for generating the permutation of an array
+in his 2013 paper, Correlated Multi-Jittered Sampling.
+
+reference:             https://graphics.pixar.com/library/MultiJitteredSampling/paper.pdf
+further comments from: https://afnan.io/posts/2019-04-05-explaining-the-hashed-permutation/
+
+*/
```

Comment:
Do you want this to be a package comment instead maybe?

Reply:
sure, wanna update it?

Reply:
yeh, it's a nice description of what this package is and where it comes from.